templates/hooks/key/sdk_read_many_post_set_output.go.tpl

```diff
@@ -0,0 +1,3 @@
+if err := rm.setResourceAdditionalFields(ctx, ko); err != nil {
```

Comment:
You'll also have to define this funciton the `hooks.go` file (e.g https://github.com/aws-controllers-k8s/lambda-controller/blob/main/pkg/resource/function/hooks.go#L511), this function will need to query the key rotation status

Reply:
This doesn't seem to work. I added in `generator.yaml` but it doesn't inject anything in `sdk.go` file. 
<img width="657" alt="image" src="https://user-images.githubusercontent.com/6952642/197578196-7ec14020-30ea-4882-92a3-698e1d347c9e.png">

<img width="651" alt="image" src="https://user-images.githubusercontent.com/6952642/197578248-1d2c5433-0f3c-4924-951b-cc0b761e1b9c.png">

<img width="614" alt="image" src="https://user-images.githubusercontent.com/6952642/197578333-a08931df-6567-4fd2-9309-a549e0afcc3d.png">

Reply:
This is pretty weird... your configuration looks correct.. if you give a wrong path to the template, does the code generator complain about it?

Reply:
> I have added to change ko.Spec in previous commit though. Will that not be sufficient?

Ideally everything should be added through code-generator.. because nexttime someone regenerate controller we'd like to keep those changes..

Reply:
@sudohikumar Also make sure that you are modifying `./generator.yaml` and not `./apis/v1lalpha1/generator.yaml`

Reply:
Hey
I validated and everything seems to be correct. But no errors there.

Reply:
Did you remember to regenerate the controller after making changes to `generator.yaml` (I only ask b/c I've done this many times myself :) )

Reply:
Hey @jaypipes ,

I did. But nothing seems to work in this case. I tried with wrong file name, but no error.

Reply:
Also, for testing, I deleted one of the existing block which was working. And it worked fine. So, I think I am doing process correctly I think.

Reply:
@jaypipes @A-Hilaly

Reply:
Thank you @jaypipes . Let me know what you find.